rbd/migration_test.go

```diff
@@ -0,0 +1,174 @@
+//go:build !(octopus || nautilus) && ceph_preview
+// +build !octopus,!nautilus,ceph_preview
+
+package rbd
+
+import (
+	"testing"
+
+	"github.com/ceph/go-ceph/rados"
```

Comment:
Style nit: put a blank line between the (local) go-ceph imports and other imports.

Reply:
Done.